mistralrs-server/src/main.rs

```diff
@@ -109,7 +117,17 @@ impl futures::Stream for Streamer {
         }
         match self.rx.try_recv() {
             Ok(resp) => match resp {
-                Response::Error(e) => {
+                Response::ValidationError(e) => {
+                    Poll::Ready(Some(Ok(Event::default().data(e.to_string()))))
```

Comment:
It looks like there is a missing call to `maybe_log_error`, could you please add that?

Reply:
That was intentional, but sure let's add it.

In general, if the user made the mistake I don't log it.

Reply:
Ah, that is a good idea. Lets not add it in that case.